src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx

```diff
@@ -85,20 +60,21 @@ class RUIIntegration extends Component {
     var self = this;
     window.ruiConfig = {
       // Custom configuration
-      embedded: false,
+      embedded: true,
       tutorialMode: false,
       homeUrl: '/donors-samples',
-      user: {
-        firstName: "Desheng",
-        lastName: "Li"
+      organ: {
```

Comment:
Will you be able to pull this config from the sample/donor config?

Reply:
It is not straightforward. I have to read the suffix from the source Display ID and parse it to the organ config. E.g TEST0036-LK => orgain: { name: "kidney", side: "left", sex: "" }. There is no way to tell what sex is.

Reply:
You can't get it from the donor block associated with the tissue? If not, please have a default (female) for when you can't do that.

Reply:
No, we don't fill in "sex" when registering a donor.